app/coffee/DiffManager.coffee

```diff
@@ -49,6 +50,8 @@ module.exports = DiffManager =
 			lastUpdate = updates[0]
 			if lastUpdate? and lastUpdate.v != version - 1
 				return callback new Error("latest update version, #{lastUpdate.v}, does not match doc version, #{version}")
+			
```

Comment:
I'm not sure what the new logic should be?

The check still applies in almost all cases, there is only the edge case of the ops being ahead of the document, and the doc version being inside in a packed op, so it's not possible to truncate the ops to the same version as the doc. This probably can't happen anymore with getting the ops first.